contentcuration/contentcuration/views/base.py

```diff
@@ -353,42 +352,54 @@ def set_language(request):
     """
```

Comment:
Could you add/replace the doc string here noting where this code came from and what you changed?